mmv1/products/orgpolicy/CustomConstraint.yaml

```
@@ -31,11 +31,19 @@ examples:
   - !ruby/object:Provider::Terraform::Examples
     name: 'org_policy_custom_constraint_basic'
     primary_resource_id: 'constraint'
+    vars:
+      policy_name: 'custom.disableGkeAutoUpgrade'
+    test_vars_overrides:
+      policy_name: 'custom.disableGkeAutoUpgrade-'
```

test_vars_overrides doesn't get the tf-test automatic replacement (and gets inserted with no quotes) - I don't remember the best practice for getting the random suffix in there offhand. Would it at all make sense to make the policy name something like `custom.disable-gke-auto-upgrade`?

```suggestion
      policy_name: '"custom.disableGkeAutoUpgrade-"'
```

---

I'm trying to preserve what I _think_ is the best practice for naming these. The documentation uses similar camel case.

I've tested the generation for the new commit, but haven't run the test (the test requires a different auth method I don't want to bother with)